src/js/components/DataSort/DataSort.js

```diff
@@ -98,15 +98,19 @@ export const DataSort = ({ drop, options, ...rest }) => {
 
   if (!drop) return content;
 
+  const tip = format({
+    id: 'dataSort.open',
+    messages: messages?.dataSort,
+  });
+
   const control = (
     <DropButton
       id={`${dataId}--sort-control`}
-      aria-label={format({
-        id: 'dataSort.open',
-        messages: messages?.dataSort,
-      })}
+      aria-label={tip}
+      tip={tip}
       kind={theme.data.button?.kind}
       icon={<Descend />}
+      hoverIndicator
```

Comment:
@jcfilben The docs say that `hoverIndicator` should only be used on plain buttons: https://v2.grommet.io/button#hoverIndicator

This isn't a plain button, so I don't think we should add it here. Also, if we have it elsewhere within Data + friends then I think we should consider removing it.

Instead, the theme button styling should drive the hover behavior.

Reply:
Good note, we do have this elsewhere in Data + Friends in the DataFilters component. I agree we shouldn't use `hoverIndicator` since they are not plain buttons and instead can use the theme to handle the hover behavior.

Reply:
Great. Should `hoverIndicator` be removed from this PR and then a separate PR can remove the instances elsewhere?

Reply:
Yes